taxi/tests/test_admin.py

```diff
@@ -0,0 +1,46 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+from django.urls import reverse
+
+from taxi.models import Car, Manufacturer
+
+
+class AdminSiteTest(TestCase):
+    def setUp(self) -> None:
+        self.client = Client()
+        self.admin_user = get_user_model().objects.create_superuser(
+            username="admin",
+            password="1a2S3r4s5t"
+        )
+        self.client.force_login(self.admin_user)
+        self.user = get_user_model().objects.create_user(
+            username="driver",
+            password="123driver123",
+            license_number="RRR54321"
+        )
+        self.manufacturer = Manufacturer.objects.create(
+            name="Ford",
+            country="USA"
+        )
+        self.car = Car.objects.create(
+            model="Focus",
+            manufacturer=self.manufacturer,
+        )

```

Comment:
I think these lines can be removed. What do you think?

Reply:
Thank you for the review. Yes you are right.